yefer-garcia/tarea-10-globant/src/pages/Quiz.jsx

```diff
@@ -0,0 +1,274 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+
+const Quiz = ({ Points, setPoints, Lifes, setLifes }) => {
+
+  const answers = ["Hombre Araña","Jurasicc Park","Batman", "La Sirenita","Mago de Oz","Libro de la Vida"] 
```

Comment:
@Yefer-Garcia , you missed the point with the components , you still repeating a lot of code